### PR TITLE
Update Git lessons

### PR DESCRIPTION
**Pull request by Lakshan-Ram**
Main changes concern the changes to GitLabs UI

### CODE REVIEW

File: git/09-pullrequests.md [hunk 1]
```diff
@@ -326,9 +326,7 @@ Web browser to your repository:
 
 > https://gitlab.cern.ch/YOUR_CERN_USERNAME/starter-kit-example
 
-Then click on the **+ (plus)** button and select **New merge request**:
-
-![New merge request](fig/gitlab-pr-newmergerequest.png)
+Then click on the **Merge Requests** button on the left hand side of the screen and select **New merge request**:
```
Comment (alexpearce) [inline git/09-pullrequests.md:329]: Are you sure you want to remove the screenshot here? If yes, you might be able to remove the image file as well (as long as it's not used elsewhere).

File: git/08-share.md [hunk 2]
```diff
@@ -178,6 +181,7 @@ What are some of the benifits of using version control, Git and GitLab?
 
 {% endchallenge %}
 
+
```
Comment (alexpearce) [inline git/08-share.md:184]: You could remove the diff in this file; the newlines probably don't change anything.

**Review by alexpearce — CHANGES_REQUESTED** [review]
Thanks a lot for refreshing this! Just a couple of comments from me.

File: git/08-share.md (hunk 2 above)
Comment (Lakshan-Ram) [reply git/08-share.md:184]: The current html website puts consecutive md lines with no spacing on the line. This means that the boxes don't render properly. 

The result looks like this 
image

I was hoping these extra spacing would solve the issue.

File: git/09-pullrequests.md (hunk 1 above)
Comment (Lakshan-Ram) [reply git/09-pullrequests.md:329]: I will replace it with a newer screenshot

File: git/08-share.md (hunk 2 above)
Comment (alexpearce) [reply git/08-share.md:184]: Ah yes. We do need the first newline you've added, before `{% solution "Solution" %}` but the rest are not needed.

Comment (Lakshan-Ram) [reply git/08-share.md:184]: Fair, I will remove those.

**Comment by chrisburr** [comment]
Thanks @Lakshan-Ram!

**Pull request merged**